### PR TITLE
[INLONG][feature][audit] add audit-common module

### LINKED ISSUE #1880: [Feature] add audit-common module[Feature]
### Description

add audit-common module

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #1880

### Motivation

add audit-common module

### CODE REVIEW

File: inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.proto [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+message Ping {
+}
+
+message Pong {
+}
+
+message AuditRequest {
+  uint64 request_id = 1;
+  AuditMessageHeader msg_header = 2;   //包头
```
Comment (aloyszhang) [inline inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.proto:44]: remove chinese annotations

File: inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java [hunk 2]
```diff
@@ -0,0 +1,6844 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: AuditApi.proto
+
+package org.apache.inlong.audit.protocol;
```
Comment (aloyszhang) [inline inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java:21]: Do we really need to upload this generated file?

Comment (baomingyu) [reply inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java:21]: yes

File: codestyle/suppressions.xml [hunk 3]
```diff
@@ -21,6 +21,7 @@
 
 <suppressions>
     <!-- suppress all checks in the generated directories -->
+    <suppress checks=".*" files="AuditApi.java" />
```
Comment (aloyszhang) [inline codestyle/suppressions.xml:24]: It's better to suppress all generated classes instead of a specified one.

**Comment by aloyszhang** [comment]
@baomingyu Pull request should respect the PULL_REQUEST_TEMPLATE

File: inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.proto (hunk 1 above)
Comment (baomingyu) [reply inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.proto]: done

File: inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java (hunk 2 above)
Comment (baomingyu) [reply inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java:21]: without this class file ，will get build error

Comment (aloyszhang) [reply inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java:21]: Why？ I think with the  .proto definition file and maven plugin commit the .proto file is enough, we can generate the target file locally

Comment (baomingyu) [reply inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java]: has delete protoc java api file ,and add build plugin in config in pom.xml

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**